### PR TITLE
Abdulla ashurov/issue272

### PR DESCRIPTION
**Pull request by abdulla-ashurov**
Add CLI and REST API integration tests for checking VID/PID in hex format

### CODE REVIEW

**Review by uabjabborov — APPROVED** [review]

File: integration_tests/cli/auth-demo-hex.sh [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+echo "Get Model with VID: $vid_in_hex_format PID: $pid_in_hex_format"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+check_response "$result" "\"productName\": \"$productName\""
+
+test_divider
+
+echo "Get $user account and confirm that alice and jack are set still present as approvers, also the sequence number should be 3 as we executed 3 txs"
```
Comment (ashcherbakov) [inline integration_tests/cli/auth-demo-hex.sh:122]: I believe all below lines are not related to this test. Account revocation has been already tested by other tests.

Comment (abdulla-ashurov) [reply integration_tests/cli/auth-demo-hex.sh]: Done! ✅

**Review by ashcherbakov — APPROVED** [review]
